src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -23,11 +23,27 @@ public abstract partial class DeriveBytes : System.IDisposable
         public abstract byte[] GetBytes(int cb);
         public abstract void Reset();
     }
+    public abstract partial class ECCurve
```

Comment:
Not yet. The code is still being extended. I realize that there are some coding conventions that I need to work on yet.